frame/election-provider-multi-phase/src/lib.rs

```diff
@@ -1533,21 +1536,27 @@ impl<T: Config> Pallet<T> {
 		<QueuedSolution<T>>::take()
 			.map_or_else(
 				|| {
-					T::Fallback::elect()
-						.map_err(|fe| ElectionError::Fallback(fe))
-						.map(|supports| (supports, ElectionCompute::Fallback))
+					T::Fallback::elect().map_err(|fe| ElectionError::Fallback(fe)).map(|supports| {
+						(supports.clone(), supports.evaluate(), ElectionCompute::Fallback)
```

Comment:
sadly this call to `.evaluate()` is pretty expensive though... let's ignore https://github.com/paritytech/substrate/pull/12069/files#r950797595 and put `None` here as the score + document in the event that the score is not available if the fallback is used.

Reply:
> sadly this call to `.evaluate()` is pretty expensive though... let's ignore https://github.com/paritytech/substrate/pull/12069/files#r950797595 and put `None` here as the score + document in the event that the score is not available if the fallback is used.

The score is available if the fallback is used due to [this](https://github.com/paritytech/substrate/pull/12069/files#diff-a8382c7a11352cf83fe1e60f06e0a9059a00443155b0a8e3189de37bc9ac4fedR1980-R1987) .

Reply:
> sadly this call to `.evaluate()` is pretty expensive though... let's ignore https://github.com/paritytech/substrate/pull/12069/files#r950797595 and put `None` here as the score + document in the event that the score is not available if the fallback is used.

Maybe `Default::default()` can be put here instead `None` due to [this](https://github.com/paritytech/substrate/blob/df0cdf2bf802af8b6dba38b7826ff19fa144324d/frame/election-provider-multi-phase/src/lib.rs#L1059-L1063) .

Reply:
I am a fan of `None` if possible instead of `Default` to emphasize there is no score associated with this.

Reply:
ok, let's put `None` here.